polyfills/IntersectionObserver/polyfill.js

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
I have no strong opinions on where or how to publish this. NPM seems like a good idea, I was just told to PR this into the WICG/IO repo :)